src/test/java/com/google/devtools/build/lib/syntax/SkylarkEvaluationTest.java

```diff
@@ -1508,7 +1508,7 @@ public void testUnionSet() throws Exception {
     new SkylarkTest("--incompatible_depset_union=false")
         .testStatement("str(depset([1, 3]) | depset([1, 2]))", "depset([1, 2, 3])")
         .testStatement("str(depset([1, 2]) | [1, 3])", "depset([1, 2, 3])")
-        .testIfExactError("unsupported operand type(s) for |: 'int' and 'int'", "2 | 4");
+        .testIfExactError("unsupported operand type(s) for |: 'int' and 'bool'", "2 | False");
```

Comment:
A shorter and clearer error message would be "unsupported operation: int | bool".

Reply:
This too I think is a bit out of the scope of this PR. I'm just correcting this test case after adding the bitwise ops (not adding/updating actual error messages in the code).

Reply:
Yes, fair enough.